docs/core/proto-docs.md

```diff
@@ -3935,6 +3953,38 @@ specified validator addr.
 
 
 
+
+<a name="osmosis.superfluid.MsgUnPoolWhitelistedPool"></a>
+
+### MsgUnPoolWhitelistedPool
+===================== MsgUnPoolWhitelistedPool
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `sender` | [string](#string) |  |  |
+| `poolId` | [uint64](#uint64) |  |  |
+| `lockId` | [uint64](#uint64) |  |  |
```

Comment:
The proposal does not state whether the msg should unpool the entire set of lockups or not. However it might be more straightforward to unpool the entire set of lockup for an account. There could be multiple locks for a single gamm denom per bonding period.

Reply:
I think adding an additional logic for partial unlockings is too over-engineering atm. For having multiple locks for a single denom per bonding period, it's not a problem since we're getting passed the lockID as well

Reply:
agreed with partial unpooling, we dont need to support it. but could we make lockID a repeated field so there can be a single transaction that unpools the entire pool?